### ISSUE TITLE
PHP Fatal errors on Atomic sites -  Uncaught WCPay\Exceptions\API_Exception

### ISSUE THREAD

**Issue by htdat** [Automattic/woocommerce-payments#5815]
### Describe the bug

Internal link - 2f856-pb

Error: 

```
PHP Fatal error:  Uncaught WCPay\Exceptions\API_Exception: Error: Accounts do not currently support `card_payments` without `transfers`. Please visit https://stripe.com/docs/connect/account-capabilities#card-payments for more details. in /wordpress/plugins/woocommerce-payments/5.6.0/includes/wc-payment-api/class-wc-payments-api-client.php:2297
Stack trace:
#0 /wordpress/plugins/woocommerce-payments/5.6.0/includes/wc-payment-api/class-wc-payments-api-client.php(2126): WC_Payments_API_Client->check_response_for_errors(Array)
#1 /wordpress/plugins/woocommerce-payments/5.6.0/includes/wc-payment-api/class-wc-payments-api-client.php(1200): WC_Payments_API_Client->request(Array, 'accounts/capabi...', 'POST', true, true)
#2 /wordpress/plugins/woocommerce-payments/5.6.0/includes/admin/class-wc-rest-payments-settings-controller.php(505): WC_Payments_API_Client->request_capability('card_payments', true)
#3 /wordpress/plugins/woocommerce-payments/5.6.0/includes/admin/class-wc-rest-payments-settings-controller.php(486): WC_REST_Payments_Settings_Controller->request_unrequested_payment_methods(Array)
#4 /wordpress/plugins/woocommerce-payments/5.6.0/includes/admin/class-wc-rest-payments-settings-controller.php(423): WC_REST_Payments_Settings_Controller->update_enabled_payment_methods(Object(WP_REST_Request))
#5 /wordpress/core/6.1.1/wp-includes/rest-api/class-wp-rest-server.php(1171): WC_REST_Payments_Settings_Controller->update_settings(Object(WP_REST_Request))
#6 /wordpress/core/6.1.1/wp-includes/rest-api/class-wp-rest-server.php(1018): WP_REST_Server->respond_to_request(Object(WP_REST_Request), '/wc/v3/payments...', Array, NULL)
#7 /wordpress/core/6.1.1/wp-includes/rest-api/class-wp-rest-server.php(442): WP_REST_Server->dispatch(Object(WP_REST_Request))
#8 /wordpress/core/6.1.1/wp-includes/rest-api.php(410): WP_REST_Server->serve_request('/wc/v3/payments...')
#9 /wordpress/core/6.1.1/wp-includes/class-wp-hook.php(308): rest_api_loaded(Object(WP))
#10 /wordpress/core/6.1.1/wp-includes/class-wp-hook.php(332): WP_Hook->apply_filters(NULL, Array)
#11 /wordpress/core/6.1.1/wp-includes/plugin.php(565): WP_Hook->do_action(Array)
#12 /wordpress/core/6.1.1/wp-includes/class-wp.php(399): do_action_ref_array('parse_request', Array)
#13 /wordpress/core/6.1.1/wp-includes/class-wp.php(780): WP->parse_request('')
#14 /wordpress/core/6.1.1/wp-includes/functions.php(1332): WP->main('')
#15 /wordpress/core/6.1.1/wp-blog-header.php(16): wp()
#16 /wordpress/core/6.1.1/index.php(17): require('/wordpress/core...')
#17 {main}
  thrown in /wordpress/plugins/woocommerce-payments/5.6.0/includes/wc-payment-api/class-wc-payments-api-client.php on line 2297
```

### To Reproduce

1. Go to '…'
2. Click on '…'
3. Scroll down to …'

### Desktop (please complete the following information):

* OS: [e.g. iOS]
* Browser [e.g. chrome, safari]
* Version [e.g. 22]

### Smartphone (please complete the following information):

* Device: [e.g. iPhone6]
* OS: [e.g. iOS8.1]
* Browser [e.g. stock browser, safari]
* Version [e.g. 22]

### Additional context

cc @shendy-a8c @cesarcosta99 as release leads, and @Automattic/gamma for having a look.

**Comment by htdat** [comment]
Update: p1679327116300659-slack-CGGCLBN58 - it's likely that Moltres will work on this one.

Updating the issue title as we are now aware that this error has happened for one month.

**Comment by MatthiasReinholz** [comment]
> Update: p1679327116300659-slack-CGGCLBN58 - it's likely that @MatthiasReinholz will work on this one.
> 
> Updating the issue title as we are now aware that this error has happened for one month.

@htdat I think you wanted to tag somebody else here. Not sure how I could help 😉

**Comment by htdat** [comment]
@MatthiasReinholz - yeah, sorry. I meant team Moltres :D

**Comment by vbelolapotkov** [comment]
One more report here: p1681234678886389-slack-CGGCLBN58

**Comment by dsmithweb-a8c** [comment]
A report of what I believe is the same issue in 7113607-zen

**Comment by naman03malhotra** [comment]
Raising awareness as there is a fresh ticket cc @daquinons

**Comment by daquinons** [comment]
> Raising awareness as there is a fresh ticket cc @daquinons

Thank you @naman03malhotra. Because I was not tagged in this issue, I didn't see any of the reports that came since the issue creation. We created https://github.com/Automattic/woocommerce-payments-server/issues/3129 which has been in the backlog, but was not prioritized until now thinking there were no new reports :) I've bumped it in our current sprint.
